src/qt/test/wallettests.cpp

```diff
@@ -164,7 +164,7 @@ void TestGUI(interfaces::Node &node) {
         int unit = walletModel.getOptionsModel()->getDisplayUnit();
         Amount balance = walletModel.wallet().getBalance();
         QString balanceComparison = BitcoinUnits::formatWithUnit(
-            unit, balance, false, BitcoinUnits::separatorAlways);
+            unit, balance, false, BitcoinUnits::separatorNever);
```

Comment:
What's this change?

Reply:
It’s weird. There’s a space separator after the 1 in the balance of 1300 now. I think this test is just broken, we need to ask ABC.